units/6_unit/05b_lesson/project.md

```diff
@@ -473,11 +474,68 @@ vice-versa) one of the values for that attribute.
 
 This is a lengthy project, and writing it entirely from scratch in the
 time available is fairly challenging. We encourage students to use
-this starter project and extend it as instructed to make a finished
+[this starter project][1] and extend it as instructed to make a finished
 project. Here is a tour of the starter project and a list of parts
 that students must implement.
 
-The scaffolding program ...
-
+The starter project contains 7 files:
+
+| File | Description |
```

Comment:
Markdown files should be readable form both in rendered form, and as a plain text file. Please format this table for those reading this in a text editor or on the command prompt.